WordPress/UITests/Tests/EditorGutenbergTests.swift

```diff
@@ -55,6 +55,33 @@ class EditorGutenbergTests: XCTestCase {
             .done()
     }
 
+    func testTextPostUndo() throws {
+
+        try BlockEditorScreen()
+            .undoIsDisabled()
+            .enterTextInTitle(text: title)
+            .addParagraphBlock(withText: content)
+            .verifyContentStructure(blocks: 1, words: content.components(separatedBy: " ").count, characters: content.count)
+            .undo()
+            .undo()
+            .verifyContentStructure(blocks: 0, words: 0, characters: 0)
+    }
+
+    func testTextPostRedo() throws {
+
+        try BlockEditorScreen()
+            .redoIsDisabled()
+            .enterTextInTitle(text: title)
+            .addParagraphBlock(withText: content)
+            .verifyContentStructure(blocks: 1, words: content.components(separatedBy: " ").count, characters: content.count)
+            .undo()
+            .undo()
+            .verifyContentStructure(blocks: 0, words: 0, characters: 0)
+            .redo()
+            .redo()
+            .verifyContentStructure(blocks: 1, words: content.components(separatedBy: " ").count, characters: content.count)
+    }
+
```

Comment:
Here we have a considerable level of duplication, `testTextPostRedo()` actually covers both tests, we would only need to add the `.undoIsDisabled()` check at the beginning. I think this is something we need to change.

Also, name wise, I'm not sure `TextPost` plays an important role here, WDYT about `testUndoRedo()`? It's totally up to you. =]

```suggestion
    func testUndoRedo() throws {

        try BlockEditorScreen()
            .undoIsDisabled()
            .redoIsDisabled()
            .enterTextInTitle(text: title)
            .addParagraphBlock(withText: content)
            .verifyContentStructure(blocks: 1, words: content.components(separatedBy: " ").count, characters: content.count)
            .undo()
            .undo()
            .verifyContentStructure(blocks: 0, words: 0, characters: 0)
            .redo()
            .redo()
            .verifyContentStructure(blocks: 1, words: content.components(separatedBy: " ").count, characters: content.count)
    }

```

Reply:
I agree! Thanks for the suggestion about unifying them and the name change, I've applied them in https://github.com/wordpress-mobile/WordPress-iOS/pull/20930/commits/33717fb3409d8c700867a9d6371441dda11b14c6